google/cloud/storage/client.py

```diff
@@ -1271,14 +1271,14 @@ def list_blobs(
             Iterator of all :class:`~google.cloud.storage.blob.Blob`
             in this bucket matching the arguments.
 
-        Example:
-            List blobs in the bucket with user_project.
+            As part of the response, you'll also get back an iterator.prefixes entity that lists object names
```

Comment:
Is there leading white space? GH is confusing me If there is please remove it, if not please ignore my comment!